### ISSUE TITLE
OutOfMemoryError

### ISSUE THREAD

**Issue by transis2** [microsoft/appcenter-sdk-android#1232]
This error happens when App Center try to send data

com.microsoft.appcenter.http.DefaultHttpClientCallTask.doHttpCall
java.lang.OutOfMemoryError: Failed to allocate a 65548 byte allocation with 37428 free bytes and 36KB until OOM

Stack traces
com.android.okio.Segment.<init> Segment.java:37
com.android.okio.SegmentPool.take SegmentPool.java:48
com.android.okio.OkBuffer.writableSegment OkBuffer.java:505
com.android.okio.OkBuffer.write OkBuffer.java:424
com.android.okio.OkBuffer.writeUtf8 OkBuffer.java:414
com.android.okio.RealBufferedSink.writeUtf8 RealBufferedSink.java:55
com.android.okhttp.internal.http.HttpConnection.writeRequest HttpConnection.java:172
com.android.okhttp.internal.http.HttpTransport.writeRequestHeaders HttpTransport.java:115
com.android.okhttp.internal.http.HttpTransport.createRequestBody HttpTransport.java:71
com.android.okhttp.internal.http.HttpEngine.sendRequest HttpEngine.java:315
com.android.okhttp.internal.http.HttpURLConnectionImpl.execute HttpURLConnectionImpl.java:399
com.android.okhttp.internal.http.HttpURLConnectionImpl.connect HttpURLConnectionImpl.java:110
com.android.okhttp.internal.http.HttpURLConnectionImpl.getOutputStream HttpURLConnectionImpl.java:221
com.android.okhttp.internal.http.DelegatingHttpsURLConnection.getOutputStream DelegatingHttpsURLConnection.java:218
com.android.okhttp.internal.http.HttpsURLConnectionImpl.getOutputStream HttpsURLConnectionImpl.java:25
com.microsoft.appcenter.http.DefaultHttpClientCallTask.doHttpCall
com.microsoft.appcenter.http.DefaultHttpClientCallTask.doInBackground
com.microsoft.appcenter.http.DefaultHttpClientCallTask.doInBackground
android.os.AsyncTask$2.call AsyncTask.java:292
java.util.concurrent.FutureTask.run FutureTask.java:237
java.util.concurrent.ThreadPoolExecutor.runWorker ThreadPoolExecutor.java:1112
java.util.concurrent.ThreadPoolExecutor$Worker.run ThreadPoolExecutor.java:587
java.lang.Thread.run Thread.java:818

**Comment by Jamminroot** [comment]
Hey, @transis2, I've tried to reproduce the problem, but wasn't able to. Tried with a device with 1gb RAM and on emulator with 384mb RAM. Both worked as expected. 
So far from logs we can only tell the user's device is running low on memory and not able to allocate any further, we assume that it is coincidence that it fails on AppCenter and not the other application.
We could use some more information on what's going on to be able to help.
